server/src/main/resources/rules/rules.js

```diff
@@ -3361,10 +3361,13 @@ var PoolType = {
 }
 
 /**
- * Namespace for determining ability to override a specific content set
- *  value
+ * Namespace for determining ability to override a specific content set value
  *
  * The list of disallowed value names are listed in this method
+ *
+ * DEPRECATED:
+ *  This functionality has been moved to ContentOverrideValidator as of Candlepin 2.5.5. This method
+ *  will no longer be called by Candlepin and is considered deprecated.
```

Comment:
This is more like a question; do we have to bump the minor version of the file even if it is a comment that changed, or not? (my intuition says no, but I just wanted to make sure)

Reply:
Yup, definitely should. The version check is used to determine if the rules are worth replacing at all. If we don't bump the version, the comment won't make it into the rules file.